search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -42,6 +42,20 @@
   [context acls concept]
   false)
 
+(defmulti esm-acls-match-concept-provider?
```

Comment:
We should use `if` over `defmulti` if there is only `if` and `else` for the choices.

Reply:
okay.